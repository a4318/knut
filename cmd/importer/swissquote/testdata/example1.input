Datum;Auftrag #;Transaktionen;Symbol;Name;ISIN;Anzahl;Stückpreis;Kosten;Aufgelaufene Zinsen;Nettobetrag;Saldo;Währung
09-10-2020 12:17:42;76396333;Kauf;VWRL;Vanguard All World ETF Dist;IE00B3RBWM25;8.0;87.60;12.90;0.00;-713.70;85.12;CHF
09-10-2020 12:13:40;00000000;Forex-Gutschrift;;;;1.0;830.07;0.00;0.00;830.07;798.82;CHF
09-10-2020 12:13:40;00000000;Forex-Belastung;;;;1.0;918.00;0.00;0.00;-918.00;0.80;USD
09-10-2020 07:02:32;00000000;Dividende;VWRL;Vanguard All World ETF Dist;IE00B3RBWM25;1.0;23.80;0.00;0.00;23.80;23.80;USD
30-09-2020 12:18:46;00000000;Depotgebühren;;;;1.0;42.27;3.25;0.00;-45.52;-31.25;CHF
30-12-2017 13:19:07;00000000;Zins;;;;1.0;0.19;0.00;0.00;0.19;418.08;USD
05-05-2015 09:05:02;00000000;Capital Gain;SYM;NAME;CH00XX;1.0;82.00;0.00;0.00;82.00;3'441.70;CHF
27-05-2020 07:02:56;00000000;Einzahlung;;;;1.0;3'656.89;0.00;0.00;3'656.89;3'656.88;USD

