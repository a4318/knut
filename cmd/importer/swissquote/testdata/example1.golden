2015-05-05 "Capital Gain SYM NAME CH00XX"
Income:Dividends  Assets:Swissquote         82 CHF (SYM)

2017-12-30 "Zins"
Income:Interest   Assets:Swissquote       0.19 USD (USD)

2020-05-27 "Einzahlung"
Expenses:TBD      Assets:Swissquote    3656.89 USD

2020-09-30 "Depotgeb√ºhren"
Assets:Swissquote Expenses:Fees          45.52 CHF ()

2020-10-09 "76396333 Kauf 8 x VWRL Vanguard All World ETF Dist IE00B3RBWM25 @ 87.6 CHF"
Assets:Swissquote Expenses:Fees           12.9 CHF (VWRL,CHF)
Assets:Swissquote Expenses:Trading       700.8 CHF (VWRL,CHF)
Expenses:Trading  Assets:Swissquote          8 VWRL (VWRL,CHF)

2020-10-09 "Dividende VWRL Vanguard All World ETF Dist IE00B3RBWM25"
Income:Dividends  Assets:Swissquote       23.8 USD (VWRL)

2020-10-09 "Forex-Gutschrift 830.07 CHF / Forex-Belastung -918 USD"
Assets:Swissquote Expenses:Trading         918 USD (CHF,USD)
Expenses:Trading  Assets:Swissquote     830.07 CHF (CHF,USD)

